leobot_description/robot/leobot.gazebo.xacro

```diff
@@ -237,4 +237,16 @@
         </sensor>
     </gazebo>
 
+    <gazebo>
+    <plugin name="object_controller" filename="libgazebo_ros_planar_move.so">
+      <commandTopic>/wheel_diff_drive_controller/cmd_vel</commandTopic>
+      <commandTopic>/teleop_twist_keyboard/cmd_vel</commandTopic>
+      <commandTopic>/joy_teleop/cmd_vel</commandTopic>
+      <odometryTopic>/mecanum_velocity_controller/odom</odometryTopic>
+      <odometryFrame>odom</odometryFrame>
+      <odometryRate>10.0</odometryRate>
+      <robotBaseFrame>FL_wheel_link</robotBaseFrame>
```

Comment:
Please created dummy link which will slide on the ground and use it as robot base frame. Check this example https://github.com/KristofRobot/frobo/blob/master/frobo_description/urdf/base.urdf.xacro#L200-L213